src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -205,7 +205,7 @@ private function findClasses(string $namespace, string $pattern, array $excludeP
             }
             // check to make sure the expected class exists
             if (!$r) {
-                throw new InvalidArgumentException(sprintf('Expected to find class "%s" in file "%s" while importing services from resource "%s", but it was not found! Check the namespace prefix used with the resource.', $class, $path, $pattern));
+                throw new InvalidArgumentException(sprintf('Expected to find class "%s" in file "%s" while importing services from resource "%s", but it was not found! Check the namespace prefix used with the resource.', $class, $path, $pattern), 0, $e ?? null);
```

Comment:
The `continue;` l204 makes me think that  `$e` always comes from the previous iteration, since this condition cannot be reached if `$e` is set, isn't it?

Reply:
Ah yes indeed, I missed that, sorry